subsys/dfu/boot/Makefile

```diff
@@ -0,0 +1,2 @@
+obj-$(CONFIG_BOOT_IMG_MANAGER) += mcuboot.o
+ccflags-$(CONFIG_BOOT_IMG_MANAGER) += -I${ZEPHYR_BASE}/subsys/dfu
```

Comment:
I believe 'ccflags' are the flags for the C compiler. C++ flags would be cppflags.